packages/fxa-auth-server/test/local/routes/totp.js

```diff
@@ -1102,19 +1335,13 @@ function setup(results, errors, routePath, requestOptions) {
     return Promise.resolve();
   });
   db.totpToken = sinon.spy(() => {
-    if (errors.totpToken) {
-      return Promise.reject(authErrors.totpTokenNotFound());
-    }
     return Promise.resolve({
-      verified:
-        typeof results.totpTokenVerified === 'undefined'
-          ? true
-          : results.totpTokenVerified,
-      enabled:
-        typeof results.totpTokenEnabled === 'undefined'
-          ? true
-          : results.totpTokenEnabled,
-      sharedSecret: secret,
+      verified: results?.totpTokenVerified || false,
```

Comment:
Just curious, is this clean up or were changes to this spy actually needed? I would not have expected that adding an mfa route would have required a change like this.